ios/fastlane/Fastfile

```diff
@@ -16,8 +16,60 @@
 default_platform(:ios)
 
 platform :ios do
-  desc "Description of what the lane does"
-  lane :custom_lane do
-    # add actions here: https://docs.fastlane.tools/actions
+
+  private_lane :apple_auth do |options|
+    ensure_env_vars(
+        env_vars: ["APP_STORE_CONNECT_API_KEY_ID", "APP_STORE_CONNECT_API_ISSUER_ID", "APP_STORE_CONNECT_API_KEY_CONTENT"]
+    )
+
+    app_store_connect_api_key(
+        key_id: ENV['APP_STORE_CONNECT_API_KEY_ID'],
+        issuer_id: ENV['APP_STORE_CONNECT_API_ISSUER_ID'],
+        key_content: Base64.decode64(ENV['APP_STORE_CONNECT_API_KEY_CONTENT'])
+    )
   end
+
+  desc "Create a release build"
+  lane :build do |options|
+    setup_circle_ci
+    version_code = options[:version_code]
+    version_name = options[:version_name]
```

Comment:
I think it would be good to add documentation that this parameters are needed, e.g.
```
  # The following parameters have to be passed:
  # version_code: The version code of the app
  # version_name: The version name of the app
```

Reply:
I think this is overhead, since it is obvious when looking at the function (second and third line of the function use the parameters), and there is a check if the parameters are missing.